watchfaces/008-contemporary-digital-rings.qml

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (C) 2018 - Timo Könnecke <el-t-mo@arcor.de>
+ *               2016 - Sylvia van Os <iamsylvie@openmailbox.org>
+ *               2015 - Florent Revest <revestflo@gmail.com>
+ *               2012 - Vasiliy Sorokin <sorokin.vasiliy@gmail.com>
+ *                      Aleksey Mikhailichenko <a.v.mich@gmail.com>
+ *                      Arto Jalkanen <ajalkane@gmail.com>
+ * All rights reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * Based on analog-precison by Mario Kicherer. Remodeled the arms to arcs
```

Comment:
If Mario Kicherer contributed to it, he should be in the Copyright list at the top of this file. Otherwise, same comment as previously.